test/jobs/parity_check/test_example.sh

```diff
@@ -1,11 +1,25 @@
 #Remove outputs from previous runs.
 rm /glade/u/home/jhreha/t-route/test/jobs/parity_check/chanobs_output/example1
 rm /glade/u/home/jhreha/t-route/test/jobs/parity_check/chanobs_output/example2
+
+#Each yaml file needs to have test outputs turned on and have a destination folder.
+# Example
+#output_parameters:
+#    test_output: ../../t-route/test/jobs/parity_check/chanobs_output/example1
+#    chanobs_output: 
+#         chanobs_output_directory: ../../t-route/test/jobs/parity_check/chanobs_output/chanobs_output_test.ncdf
+
 #Creates master branch outputs to use for benchmark.
-git checkout master #master
+# git checkout master #master
 python -m nwm_routing -f -V3 parity_florence_example1.yaml 
```

Comment:
we will want some comments that tell the user to specify the which yaml file to run here. Maybe replace the existing .ymal file name with a dummy placeholder like <test.yaml>.